guides/common/modules/proc_configuring-the-project-mcp-server.adoc

```diff
@@ -0,0 +1,40 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-the-{project-context}-mcp-server"]
+= Configuring the {Project} MCP server
+
+Deploy the {Project} MCP server as a container and make it accessible to your MCP client.
+
+.Procedure
+. Optional: Pull the latest version of the MCP container image from the registry:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman pull {mcp-server-image}
+----
+. Run the container, specifying its connection to your {ProjectServer}.
+For example:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman run --interactive --tty --publish localhost:8080:8080 \
+  {mcp-server-image} \
+  --foreman-url https://_{foreman-example-com}_ \
+  --host 0.0.0.0 \
+  --port 8080 \
+  --transport stdio
```

Comment:
This will work only if `FOREMAN_USERNAME` and `FOREMAN_PASSWORD` environment variables are set (`export FOREMAN_USERNAME=<usr>`, `export FOREMAN_PASSWORD=<pwd>`), which is probably the recommended way.  Otherwise the `--foreman-username` and `--foreman-password` options will be needed for for stdio transport.